### PR TITLE
Disable some flaky checks in the test.

### PR DESCRIPTION
**Pull request by spershin**
Test plan - (Please fill in how you tested your changes)

Please make sure your submission complies with our [Development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [Formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), and [Commit Message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests) guidelines. Don't forget to follow our [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution) for any code copied from other projects.

Fill in the release notes towards the bottom of the PR description.
See [Release Notes Guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) for details.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-tests/src/test/java/com/facebook/presto/server/TestQueryResource.java [hunk 1]
```diff
@@ -96,12 +96,14 @@ public void testGetQueryInfos()
 
         infos = getQueryInfos("/v1/query?limit=5");
         assertEquals(infos.size(), 5);
-        assertEquals(infos.get(0).getQuery(), "SELECT * from tpch.sf100.orders -- 1");
-        assertEquals(infos.get(1).getQuery(), "SELECT * from tpch.sf100.orders -- 2");
-        assertEquals(infos.get(2).getQuery(), "SELECT * from tpch.sf100.orders -- 3");
-        assertEquals(infos.get(3).getQuery(), "SELECT 3");
-        assertEquals(infos.get(4).getQuery(), "SELECT x FROM y");
-        assertStateCounts(infos, 0, 1, 3, 1);
+        // It seems that reliance on the sleep and query timing don't work together too well.
```
Comment (ajaygeorge) [inline presto-tests/src/test/java/com/facebook/presto/server/TestQueryResource.java:99]: I think it is best to disable the entire test .
Running locally I can see failures as well on line 92 above.

**Review by ajaygeorge — CHANGES_REQUESTED** [review]
I think we need to disable the entire test. Test locally fails at an earlier point for me.

**Comment by spershin** [comment]
> I think we need to disable the entire test. Testing locally fails at an earlier point for me.

How do I disable java test properly?

**Comment by ajaygeorge** [comment]
> > I think we need to disable the entire test. Testing locally fails at an earlier point for me.
> 
> How do I disable java test properly?

Add `enabled = false` in the `@Test`

File: presto-tests/src/test/java/com/facebook/presto/server/TestQueryResource.java (hunk 1 above)
Comment (spershin) [reply presto-tests/src/test/java/com/facebook/presto/server/TestQueryResource.java:99]: Local runs fail for me all the time, even before the last change.
I think it is a very flaky test.

**Review by ajaygeorge — APPROVED** [review]
LGTM

**Pull request merged**